app/models/concerns/type_of_means_assessment.rb

```diff
@@ -47,30 +48,30 @@ def evidence_of_passporting_means_forthcoming?
   # blocked until the NINO or benefit evidence is provided.
   def nino_forthcoming?
     return false unless has_passporting_benefit?
-    return false unless applicant.has_nino == 'no'
-    return false if applicant.will_enter_nino == 'yes'
+    return false unless benefit_check_recipient.has_nino == 'no'
+    return false if benefit_check_recipient.will_enter_nino == 'yes'
 
-    applicant.will_enter_nino == 'no' || kase.is_client_remanded == 'yes'
+    benefit_check_recipient.will_enter_nino == 'no' || kase.is_client_remanded == 'yes'
```

Comment:
I'm wondering if it would be better to have a dedicated object for the benefit check questions instead of referring to a person.

This way, we can ask questions like benefit_check.nino_forthcoming?. The logic to understand who the check is on could also live there instead of within the crime application.